homeassistant/components/lutron/entity.py

```diff
@@ -41,6 +45,10 @@ def unique_id(self) -> str | None:
             return None
         return f"{self._controller.guid}_{self._lutron_device.uuid}"
 
+    def update(self) -> None:
+        """Update the entity's state."""
+        self._update_attrs()
```

Comment:
I think we should keep the difference between the subscription callback that just uses the cached device state and the `update` method which should force a refresh of state before updating attributes.

The library uses `last_state` for the former and `state` for the latter, same for `level` etc.